--- hyprlauncher-0.1.5/src/config/ConfigManager.cpp.orig	2026-02-04 14:14:02.167637934 -0500
+++ hyprlauncher-0.1.5/src/config/ConfigManager.cpp	2026-02-04 14:14:52.694620281 -0500
@@ -3,6 +3,8 @@
 #include "../helpers/Log.hpp"
 
 #include <hyprutils/path/Path.hpp>
+#include <climits>
+#include <unistd.h>
 
 CConfigManager::CConfigManager() : m_inotifyFd(inotify_init()) {
     const auto CFGPATH = Hyprutils::Path::findConfig("hyprlauncher").first.value_or("");
--- hyprlauncher-0.1.5/src/finders/desktop/DesktopFinder.cpp.orig	2026-02-04 14:31:04.984483548 -0500
+++ hyprlauncher-0.1.5/src/finders/desktop/DesktopFinder.cpp	2026-02-04 14:31:56.094942806 -0500
@@ -10,6 +10,7 @@
 #include <sys/inotify.h>
 #include <sys/poll.h>
 #include <unordered_set>
+#include <unistd.h>
 
 #include <hyprutils/string/String.hpp>
 #include <hyprutils/os/Process.hpp>
